### DISCUSSION TITLE
Export Faces / write to exif?

### DISCUSSION THREAD

**Discussion by pcace** [photoprism/photoprism#2052] [Q&A] [answered]
Hi, 
is there any way to export found faces somehow or write them directly to exif data? 
Where is the information about found Faces stored?

Cheers and Thanks!

**Comment by graciousgrey** [comment, marked as answer]
PhotoPrism does not yet write metadata back to exif or xmp files.

It's planned see:
https://github.com/photoprism/photoprism/issues/402
https://github.com/photoprism/photoprism/discussions/1092

At the moment most metadata is written to yml export files (https://docs.photoprism.app/user-guide/advanced/export/).
Faces will be added to those exports as well --> https://github.com/photoprism/photoprism/issues/1555
